### PR TITLE
WGLC updates 3/x (Neil's stuff)

### PR DESCRIPTION
**Pull request by bc-pi**
https://mailarchive.ietf.org/arch/msg/oauth/zQ7Q0cwuwiAJOMJoYTols_oR4W4/

### CODE REVIEW

**Review by danielfett — APPROVED** [review]
Danke!

File: draft-ietf-oauth-selective-disclosure-jwt.md [hunk 1]
```diff
@@ -524,8 +522,8 @@ alphanumerically or randomly, after potentially adding
 decoy digests as described in (#decoy_digests). The precise method does not matter as long as it
 does not depend on the original order of elements.
 
-For example, using the digest of the object property Disclosure created above,
-the Issuer could create the following SD-JWT payload to make `given_name`
+For example, using the digest of the Disclosure from (#hashing_disclosures),
+the Issuer could create the following SD-JWT payload to make `family_name`
```
Comment (Sakurann) [inline draft-ietf-oauth-selective-disclosure-jwt.md:526]: the value on line 534 should also be changed to `X9yH0Ajrdm1Oij4tWso9UzzKJvPoDxwmuEcO3XAdRC0`, no?

Comment (bc-pi) [reply draft-ietf-oauth-selective-disclosure-jwt.md:526]: yup, i believe you are correct. good catch, thx

Comment (bc-pi) [reply draft-ietf-oauth-selective-disclosure-jwt.md:526]: and fixed 5bec8b4a13fdfdd7f1edfd014ae29b343b75839b

**Review by Sakurann — APPROVED** [review]

**Pull request merged**